pkg/kubelet/server.go

```diff
@@ -148,7 +202,8 @@ func (s *Server) InstallDefaultHandlers() {
 		Operation("getPods"))
 	s.restfulCont.Add(ws)
 
-	s.restfulCont.Handle("/stats/", &httpHandler{f: s.handleStats})
+	s.restfulCont.HandleWithFilter("/stats/", &httpHandler{f: s.handleStats})
```

Comment:
changed s.restfulCont to an interface, wrapped the container so all Handle calls delegate to HandleWithFilter, and keep track of registered handled paths so the tests don't have to hard-code paths